extensions/default/src/ViewerLayout/index.tsx

```diff
@@ -1,11 +1,12 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useCallback } from 'react';
 import PropTypes from 'prop-types';
 import { useNavigate } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 import { useLocation } from 'react-router';
 
 import {
   SidePanel,
+  SidePanelWithService,
```

Comment:
SidePanelWithService should live in default

Reply:
Moved to default extension